content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -58,6 +58,61 @@ corresponding reclamation or reconciliation process, as a given admission
 controller does not know for sure that a given request will pass all of the
 other admission controllers.
 
+The ordering of these calls can be seen below.
+
+
+{{< mermaid >}} 
+
+
+%% Have to set fontSize globally as this isn't respected due to https://github.com/mermaid-js/mermaid/issues/3990
+%%{init:{"theme":"neutral","fontSize": "22px", "sequence": {"mirrorActors":true},
+    "themeVariables": {
+        "actorBkg":"royalblue",
+        "actorTextColor":"white",
+        "loopTextColor": "royalblue",
+        "noteBkgColor": "grey",
+        "fontSize": "22px"
+}}}%%
```

Comment:
- we'd prefer to use Kubernetes colors
- we'd rather not customize the text size

If that's not feasible, you could export the Mermaid diagram as SVG and use that instead.

More tips: https://www.k8s.dev/blog/2021/12/01/improve-your-documentation-with-mermaid.js-diagrams/

Reply:
See https://github.com/kubernetes/website/pull/43836#issuecomment-2036576409